api/controllers/swagger.go

```diff
@@ -33,7 +33,7 @@ func (s *SwaggerController) ServeSwaggerSpec(c *fireball.Context) (fireball.Resp
 	spec := swagger.Spec{
 		SwaggerVersion: "2.0",
 		Host:           c.Request.Host,
-		Schemes:        []string{"https"},
+		Schemes:        []string{"http", "https"},
```

Comment:
[nit] I made this change as well, but i think we should let this be configureable via env vars or something... pretty big security risk to have swagger open via HTTP if we forget to remove this part

Reply:
Yeah, great point 👍

✅